src/index.html

```diff
@@ -2,11 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    rel="stylesheet"
+    href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+
+      <div class="calendar calendar__month--length--29">
+        <span class="calendar__day day day--1 day__start--mon"></span>
```

Comment:
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/1ed123e0-a91b-41b9-8c21-43cbbe4cabd7)
```suggestion
      <div class="calendar calendar__month-length--29 calendar__start-day--mon">
        <span class="calendar__day day day--1"></span>
```